src/AutomatedCar/Helpers/GearEnum.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace AutomatedCar.Helpers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    public enum Gear
+    {
+        Park,
+        Reverse,
+        Neutral,
+        Drive
+    }
+
+    public enum Shifts
+    {
+        One,
```

Comment:
We should include a "None" value as well, because otherwise the CurrentShift property in GearShift.cs will give a false value when the CurrentGear property is in any other state than "Drive".